tags/resources/toc-supporting-guides/tech-papers.md

```diff
@@ -0,0 +1,155 @@
+
+#### Executive Summary and Content Wrap-up
+
+If the paper exceeds five pages, write an executive summary no more than three paragraphs. Executive summaries are intended to be somewhat generalized and provide an overview of the paper without diving into the details. It allows readers to ascertain relevance of the content before committing to read it in entirety to ascertain value. Once complete, finalize content and discussions internal to the group.
+
+#### Narrative Voice
+
+Assign no more than three individuals for this task to ensure a unified voice. Ensure consistent language, phrasing, acronym use, footnotes, and citations. The final result of this unification should ensure the paper reads consistently, like it was written by a single entity. This also functions as the final edit before group review.
+
+#### Final Group Review
+
+The group collectively reviews the polished paper before opening it for community review. This is the last cursory check before opening to public content. It is imperative that contributors verify all referencable items are linked, cited, concepts explained, acronyms spelled out, and any unclear content is corrected. If the group fails to gain consensus on any of the content presented they may choose to remove it or explain the different perspectives that allow the reader to draw their own conclusions, this is paritcularly important where a technical recommendation results in security or performance trade offs.
+
+The group should assign individuals to serve as comment adjudicators for public review at this point. These individuals will be responsible for validating commentary against the defined scope, assumptions, and audience.
+
+#### Community Review
+
+When ready for commmunity review, lock the paper from editing (except for Adjudicators). Make it public for comments and suggestions. Announce a two week public comment period via the TAG's mailing list as well as the TOC's mailing list. TAGs and groups may wish to leverage a git workflow for this, by moving the paper to markdown and filing a PR which would allow public comments on the PR.
+
+#### Public Comment Adjudication
+
+Adjudicators are may choose to resolve community comments and suggestions as they are applied or may wait for the full comment period to close before adjudicating. Comments resulting in larger discussions should be decisively resolved and their resolution documented clearly, particularly if the comment concerns will not be included in the final paper. Adjudicators should make reasonable effort to ensure the comments and concerns raised are duly considered and incorporated, as appropriate.
+
+#### Leadership Review and Approval
+
+Once public comments are adjudicated, to ensure appropriate oversight on content, two additional reviews are required before the paper may be published.
```

Comment:
who provides these additional reviews? Is the TAG sponsor one of the two?

Reply:
added below